### ISSUE TITLE
tortoise git 으로 브랜치 비교

### ISSUE THREAD

**Issue by MinSikMoon** [MinSikMoon/links#108]
http://becdetat.com/compare-branches-in-tortoise-git.html
07 Aug 2014
Using a PR / peer review model when developing in a team is a great way keep quality high with a large number of changes. This model relies on being able to review diffs easily, something which Bitbucket is usually pretty good at.

Unfortunately Bitbucket falls down once PRs reach a certain size:

This merge is too large to display

i haz a sadz

Ideally, PRs should be small enough that this doesn’t crop up, but when you’re performing large refactors or working on significant feature branches the ideal isn’t always possible.

TortoiseGit to the rescue: Compare (Diff) branches in Tortoise Git, or how to preview changes before doing a merge.

Basically:

shift-right click the folder
Select TortoiseGit -> Browse Reference
Select the two branches to compare using control (usually the current branch and master)
Right-click and select Compare selected refs
